tests/sentry/workflow_engine/migration_helpers/test_migrate_rule.py

```diff
@@ -97,13 +109,27 @@ def test_create_issue_alert_no_actions(self):
             },
             condition_result=True,
         ).exists()
-        assert DataConditionGroupAction.objects.filter(condition_group=if_dcg).count() == 0
 
-    def test_update_issue_alert(self):
-        # TODO: update after action registry is merged
-        pass
+        dcg_actions = DataConditionGroupAction.objects.get(condition_group=if_dcg)
+        action = dcg_actions.action
+        assert action.type == Action.Type.SLACK  # tested fully in test_migrate_rule_action.py
 
-    def test_update_issue_alert_no_actions(self):
+    def test_create_issue_alert_detector_exists(self):
+        self.create_detector()
+        migrate_issue_alert(self.issue_alert, self.rpc_user)
+
+        # does not create a new error detector
+
+        detector = Detector.objects.get(project_id=self.project.id)
+        assert detector.name == "Error Detector"
```

Comment:
I'm a little confused by lines 121 and 124 - is this supposed to be an error detector or not?

Reply:
migrating an issue alert for a project should only create an error detector if you don't already have one

Reply:
later i will add creating this detector by default upon project creation

Reply:
hmmm then maybe the test could assert that it can't find an error detector for the given project before calling `migrate_issue_alert`?

Reply:
`self.create_detector()` creates the detector for the project, maybe i just assert that getting the detector after calling `migrate_issue_alert` is the same as the one i created before migrating the issue alert?